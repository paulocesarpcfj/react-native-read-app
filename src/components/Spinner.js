import React from 'react';
import { View, ActivityIndicator } from 'react-native';

const spinnerStyle = {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
};

export default () => (
    <View style={spinnerStyle}>
        <ActivityIndicator size="large" style={{ paddingVertical: '50%' }} />
    </View>
);
