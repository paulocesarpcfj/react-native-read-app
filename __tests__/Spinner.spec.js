import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../src/components/Spinner';

it('renders correctly', () => {
    const spinner = renderer.create(
        <Spinner />
    ).toJSON();
    expect(spinner).toMatchSnapshot();
});
