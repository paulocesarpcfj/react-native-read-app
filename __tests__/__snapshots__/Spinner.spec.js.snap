// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
    style={
      Object {
        "paddingVertical": "50%",
      }
    }
  />
</View>
`;
